### PR TITLE
hash table

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/hashTable/HashTable.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    this.map = new Node[size];
+  }
+  
+  private int hash(K key) {
+    int hashValue = 0;
+    char[] letters = key.toString().toCharArray();
+    for (char letter : letters) {
+      hashValue += letter;
+    }
+    hashValue = (hashValue * 5999) % map.length;
```
Comment (evanslaton) [inline code401challenges/src/main/java/hashTable/HashTable.java]: I think you meant 599 (5999 isn't a prime number)
